packages/webapp/app/helloworld/client/client.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
```

Comment:
We need `helloworld`?

Reply:
Yeah no problem, maybe put robots disallow so it does not get scraped accidentally

Reply:
Added noindex ✔️